src/graph/CreateEdgeIndexExecutor.cpp

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/CreateEdgeIndexExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+CreateEdgeIndexExecutor::CreateEdgeIndexExecutor(Sentence *sentence,
+                                                 ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<CreateEdgeIndexSentence*>(sentence);
+}
+
+Status CreateEdgeIndexExecutor::prepare() {
+    return Status::OK();
+}
+
+void CreateEdgeIndexExecutor::execute() {
+    auto status = checkIfGraphSpaceChosen();
+    if (!status.ok()) {
+        DCHECK(onError_);
+        onError_(std::move(status));
+        return;
+    }
+
+    auto *mc = ectx()->getMetaClient();
+    auto *name = sentence_->indexName();
```

Comment:
`const` is better?

Reply:
I don't know what's the practical use ?

Reply:
> I don't know what's the practical use ?
Actually it's just my code like, here's what I think ：
Reason 1 : The method `sentence_->indexName()` return value is a const string *. 
Reason 2 : This value is an immutable constant.

Reply:
I think the index name is immutable. The compiler maybe have some optimization for the `const auto*` .